tap_zendesk/streams.py

```diff
@@ -314,33 +347,32 @@ def emit_sub_stream_metrics(sub_stream):
                 try:
                     for audit in audits_stream.sync(ticket["id"]):
                         self._buffer_record(audit)
-                except HTTPError as e:
-                    if e.response.status_code == 404:
-                        LOGGER.warning("Unable to retrieve audits for ticket (ID: %s), record not found", ticket['id'])
-                    else:
-                        raise e
-
+                except http.ZendeskNotFoundError:
+                    # Skip stream if ticket_audit does not found for particular ticekt_id. Earlier it throwing HTTPError
+                    # but now as error handling updated, it throws ZendeskNotFoundError.
+                    message = "Unable to retrieve audits for ticket (ID: {}), record not found".format(ticket['id'])
```

Comment:
@prijendev Don't use `format()`

Reply:
We they have migrated to the latest version. That's why `format()` has been kept.